src/app/[locale]/(app)/settings/_components/Languages.tsx

```diff
@@ -112,7 +112,7 @@ function LocaleSwitcher({
                     as="h3"
                     className="text-lg font-bold text-white"
                   >
-                    Switch language
+                    {t("settings.languages.switch")}
```

Comment:
You need to import `t` in this component as well

Reply:
Thanks ! Yes, when repeating things I may forget this 😅

Reply:
After looking at it it's already the case 🤔

Reply:
https://github.com/dumpus-app/dumpus-app/blob/02a4d27cc34f86e6945d1aefbeb130e022d9ac0c/src/app/%5Blocale%5D/(app)/settings/_components/Languages.tsx#L6